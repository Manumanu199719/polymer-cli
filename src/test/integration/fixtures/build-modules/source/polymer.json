{
  "builds": [
    {
      "name": "just-compiled",
      "js": {
        "compile": true
      }
    }
  ]
}
