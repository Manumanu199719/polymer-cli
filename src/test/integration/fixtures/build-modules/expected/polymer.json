{
  "entrypoint": "index.html",
  "fragments": [],
  "sources": [
    "src/**/*",
    "index.html"
  ],
  "extraDependencies": [
    "bower_components/webcomponentsjs/*.js"
  ],
  "builds": [
    {
      "name": "just-compiled",
      "js": {
        "compile": true
      }
    }
  ]
}